notebook/static/notebook/less/cell.less

```diff
@@ -21,39 +27,52 @@ div.cell {
     /* This acts as a spacer between cells, that is outside the border */
     margin: 0px;
     outline: none;
+    
+    position: relative;
+    overflow: visible;
 
-    ._selected_style(transparent, transparent, @cell_border_width);
-
-    &.jupyter-soft-selected {
-        border-left-color: @selected_border_color_light;
-        border-left-color: @soft_select_color;
-        border-left-width: @cell_border_width;
-        padding-left: @cell_padding - @cell_border_width;
-        border-right-color: @soft_select_color;
-        border-right-width: @cell_border_width;
-        background: @soft_select_color;
-
-        @media print {
-            border-color: transparent;
-        }
+    &:before {
+        ._selected_style(transparent);
     }
 
+    // &.jupyter-soft-selected {
```

Comment:
If we no longer need this chunk, let's get rid of it. If we're not sure, let's work out who to ask.